server/routes/surveyRoutes.js

```diff
@@ -1,14 +1,15 @@
 const _ = require('lodash');
-const Path = require('path-parser');
-const { URL } = require('url');
 const mongoose = require('mongoose');
+
 const requireLogin = require('../middlewares/requireLogin');
 const requireCredits = require('../middlewares/requireCredits');
 const Mailer = require('../services/Mailer');
 const surveyTemplate = require('../services/emailTemplates/surveyTemplate');
-
+const SurveyService = require('../service/survey.service.js');
```

Comment:
This is my new service, notice my file naming convention of `survey.service.js`. It's all lowercase, and has a `.type` naming convention, that way you know what type of object your dealing with. I kept it in the services folder for now, but as I mentioned you should group by feature, it's just not worth doing at this point on this small of a project.